app/controllers/organizations/adopter_fosterer/adopted_pets_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Organizations
+  module AdopterFosterer
+    class AdoptedPetsController < Organizations::BaseController
+      before_action :context_authorize!
+      layout "adopter_foster_dashboard"
+
+      def index
+        @adopted_pets = authorized_scope(AdopterApplication.joins(:pet)
+        .where(status: :adoption_made, profile_show: true), with: Organizations::AdopterFosterAdoptedPetPolicy)
+      end
+
+      def files
```

Comment:
We could create a new route and controller to handle this instead e.g., 
```
module Organizations
  module AdopterFosterer
    class AdoptedPetFilesController
```

And use the index action to get the list of files for a given pet. This will keep things RESTful so we don't have to use custom methods. What do you think?

Reply:
Nice.. I liked this approach to keep things more RESTful. Have made the required changes and created new files_controller.rb